spec.html

```diff
@@ -181,19 +181,29 @@ <h1>RealmImportValue ( _specifierString_, _exportNameString_, _callerRealm_, _ev
 				1. Assert: Type(_exportNameString_) is String.
 				1. Assert: _callerRealm_ is a Realm Record.
 				1. Assert: _evalRealm_ is a Realm Record.
-				1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
+				1. Let _innerCapability_ be ! NewPromiseCapability(%Promise%).
 				1. Let _runningContext_ be the running execution context.
 				1. If _runningContext_ is not already suspended, suspend _runningContext_.
 				1. Push _evalContext_ onto the execution context stack; _evalContext_ is now the running execution context.
-				1. Perform ! HostImportModuleBindingDynamically(_callerRealm_, _evalRealm_, _specifierString_, _exportNameString_, _promiseCapability_).
+				1. Perform ! HostImportModuleDynamically(*null*, _specifierString_, _innerCapability_).
```

Comment:
I'd suggest extending `HostImportModuleDynamically` to accept a realm argument:

```suggestion
				1. Perform ! HostImportModuleDynamically(*null*, _specifierString_, _evalRealm_, _innerCapability_).
```

Then to update the spec text for `HostImportModuleDynamically` to note that all module records should have their realm set to this realm. That seems a nice generalization as well.

Reply:
That's great, but involves circling back with the ECMA-262 editors. I'm moving this to a new issue to avoid losing track of it.